app-backend/api/src/main/scala/user/Routes.scala

```diff
@@ -42,6 +51,11 @@ trait UserRoutes extends Authentication
       pathEndOrSingleSlash {
         get { getUserByEncodedAuthId(authIdEncoded) } ~
         put { updateUserByEncodedAuthId(authIdEncoded) }
+      } ~
+      pathPrefix("dropbox-setup") {
```

Comment:
I was thinking it would be along the same lines of `/me`, since that's also under `/users`. But I guess it isn't really a user resource, while `/me` is more of an alias